packages/components/inputs/src/checkbox/Checkbox.tsx

```diff
@@ -6,21 +6,9 @@ import { ControlledInput, ControlledInputProps } from '..';
 export interface CheckboxProps extends ControlledInputProps {}
 
 export const Checkbox = ({
-  disabled = false,
-  required = false,
   testId = 'cf-ui-checkbox',
   type = 'checkbox',
```

Comment:
I don't think this needs to be changable here.

Reply:
upps, you're right